python/clicastInterface.py

```diff
@@ -0,0 +1,249 @@
+import os
+import re
+import shutil
+import subprocess
+import sys
+import time
+
+from vector.lib.core.system import cd
+
+
+commandFileName = "commands.cmd"
+globalClicastCommand = ""
+
+
+enviroNameRegex = "-e([^\s]*)"
+
+
+def getEnviroPathFromCommand(command):
+
+    # TBD in the future we will change the caller to pass in the environment path
+    # No error handling because the caller will guarantee that we have a valid command
+    match = re.search(enviroNameRegex, command)
```

Comment:
Who generates what's passed to this? It is valid to do `-e<space><env name>`.

The regex could be:

```py
enviroNameRegex = "-e\s*([^\s]*)"
```